frankenphp.go

```diff
@@ -242,6 +246,39 @@ func Config() PHPConfig {
 	}
 }
 
+// MaxThreads is the maximum number of threads that can be started, it is set by frankenphp.Init and should be considered read-only.
+var MaxThreads int
```

Comment:
Does this variable has to be exported?

Reply:
It's used in tests to verify things (this is how I discovered test configurations were leaking into later tests). It might be useful to prevent regressions, but it isn't read by FrankenPHP, it is just written from FrankenPHP.

Reply:
Maybe could we mark it as "internal" or "experimental" then?

Reply:
Good idea!